robot/EDA/resources/CourseConnectionsSettingsPageObject.py

```diff
@@ -20,3 +22,62 @@ def _is_current_page(self):
             locator_tab,
             error=f"Course Connections tab with locator '{locator_tab}' is not available on the page"
         )
+
+    def set_enable_course_connections(self):
+        """ Set the checkbox for 'Enable Course Connections' field """
+        locator = eda_lex_locators["eda_settings_cc"]["enable_cc_checkbox"]
+        self.selenium.wait_until_page_contains_element(locator)
+        self.selenium.click_element(locator)
+        time.sleep(0.5)
+
+    def update_enable_cc_to_default(self):
+        """ Updating the `Enable Course Connections` checkbox to default value (false)"""
+        locator_default = eda_lex_locators["eda_settings_cc"]["default_cc_checkbox"]
+        locator_edit_mode = eda_lex_locators["eda_settings_cc"]["enable_cc_checkbox"]
+
+        self.selenium.wait_until_page_contains_element(locator_default)
+        actual_value = self.selenium.get_webelement(locator_default).get_attribute("alt")
+        if not actual_value == "False":
+            self.eda.click_edit_on_eda_settings_page()
+            self.selenium.wait_until_page_contains_element(
+                locator_edit_mode, error=f"Enable course connections checkbox with locator '{locator_edit_mode}' is not available on the page")
+            self.selenium.click_element(locator_edit_mode)
+            self.eda.click_action_button_on_eda_settings_page("Save")
+
+    def verify_dropdown_values(self, field, *args):
```

Comment:
Try to see if you can enhance this keyword to handle subdictionaries as you can send multiple criteria to validate instead of calling the same keyword twice